test/integration_test/migration_test.go

```diff
@@ -27,7 +27,7 @@ import (
 
 const (
 	migrationRetryInterval = 10 * time.Second
-	migrationRetryTimeout  = 5 * time.Minute
+	migrationRetryTimeout  = 10 * time.Minute
```

Comment:
we also wanted to increase groupcloudsnapshot timeouts?

Reply:
Fixed.